library/blob.c

```diff
@@ -2049,10 +2110,11 @@ int eblob_write_prepare(struct eblob_backend *b, struct eblob_key *key,
 			"key: %s, size: %" PRIu64 ", flags: %s",
 			eblob_dump_id(key->id), size, eblob_dump_dctl_flags(flags));
 
+
 	/* Sanity */
 	if (b == NULL || key == NULL) {
 		err = -EINVAL;
-		goto err_out_exit;
+		return err;
```

Comment:
This change drops log with `eblob_write_prepare: finished`. Why?

Reply:
returned